pyscriptjs/tests/integration/test_script_type.py

```diff
@@ -1,8 +1,10 @@
-from .support import PyScriptTest, skip_worker
+import pytest
+
+from .support import PyScriptTest
 
 
 class TestScriptTypePyScript(PyScriptTest):
-    @skip_worker("FIXME: js.document")
+    @pytest.mark.skip("FIXME: display() without target is broken")
```

Comment:
this test works fine too:

```html
<!doctype html>
<html>
  <head>
    <script type="module" src="https://cdn.jsdelivr.net/npm/@pyscript/core"></script>
  </head>
  <body>
    <script type="py">
      from pyscript import display

      display('hello\nworld')
    </script>
  </body>
</html>
```

although I've spotted an issue if the script is in the head instead of the body ... will fix this (yet this code works)

Reply:
Update: the MR is here https://github.com/pyscript/pyscript/pull/1658/files#diff-7a3bed362bc676d12ed6963d31fe4fda473262e3090a49b333702731f5c8d916